pym/bob/input.py

```diff
@@ -576,6 +577,67 @@ def isDeterministic(self):
     def hasJenkinsPlugin(self):
         return True
 
+    def callGit(self, workspacePath, *args):
+        cmdLine = ['git']
+        for arg in args:
+            cmdLine.append(arg)
+        proc = subprocess.Popen(cmdLine, cwd=os.path.join(os.getcwd(),workspacePath, self.__dir),
+                universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = proc.communicate()
```

Comment:
Please don't throw away stderr. It might be valuable to the user. You may have a look at `subprocess.check_output()`. That should be better suited thatn `Popen` for the job I guess.